bot/interact.go

```diff
@@ -51,12 +51,21 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 		case slack.InteractionTypeDialogSubmission:
 			switch message.CallbackID {
 			case "report_message":
-				msg := fmt.Sprintf("<@%s> sent a message report:\n- *Reason*: %s\n- *Feeling Scale*: %s of 5\n%s",
-					message.User.Name,
-					message.Submission["reason"],
-					message.Submission["scale"],
-					message.State,
-				)
+				var msg string
+				if message.Submission["anonymous"] == "anonymous" {
```

Comment:
You could simplify all this code by just making the user, including the `<@name>` part of a var right?
E.g. 

```go
if message.Submission["anonymous"] == "anonymous" {
  user = "anonymous"
} else {
  user = fmt.Sprintf("<@%s>", message.User.Name)
}
```

Reply:
You are right, done